build.sbt

```diff
@@ -12,7 +12,7 @@ lazy val root = (project in file(".")).
     inThisBuild(List(
       version := "0.3.0-SNAPSHOT",
       organization := "org.scala-sbt",
-      crossScalaVersions := Seq("2.12.2", "2.11.11", "2.10.6"),
+      crossScalaVersions := Seq("2.12.3", "2.11.11", "2.10.6"),
```

Comment:
Btw, this isn't reflected in .travis.yml or CONTRIBUTING.md.

Btw^2 we should switch this repo to use https://github.com/dwijnand/sbt-travisci :)